app/classifier/tasks/slider/index.jsx

```diff
@@ -0,0 +1,179 @@
+import React from 'react';
+import GenericTask from '../generic';
+import SliderTaskEditor from './editor';
+import { Markdown } from 'markdownz';
+
+const NOOP = Function.prototype;
+
+class Summary extends React.Component {
+  render() {
+    let answer = 'No answer';
+    if (this.props.annotation.value !== null) {
+      answer = this.props.annotation.value;
+    }
+    return (
+      <div>
+        <div className="question">
+          {this.props.task.instruction}
+        </div>
+        <div className="answer">
+          {answer}
+        </div>
+      </div>
+    );
+  }
+}
+Summary.displayName = 'SliderSummary';
+
+Summary.propTypes = {
+  task: React.PropTypes.shape(
+    {
+      instruction: React.PropTypes.string
+    }
+  ).isRequired,
+  annotation: React.PropTypes.shape(
+    {
+      value: React.PropTypes.string
+    }
+  ).isRequired
+};
+
+class SliderTask extends React.Component {
+  constructor(props) {
+    super(props);
+    this.task = Object.assign(
+      {
+        help: '',
+        instruction: '',
+        required: false,
+        max: 1,
+        min: 0,
+        step: 0.1,
+        defaultValue: 0
+      },
+      this.props.task
+    );
+    let value = this.task.defaultValue;
+    if (this.props.annotation.value !== null) {
+      value = this.props.annotation.value;
+    }
+    this.state = { value };
+    this.handleChange = this.handleChange.bind(this);
+  }
+  handleChange() {
```

Comment:
An `onChange` callback is passed an event as an argument.  This event can be used to get the value the `input` element is changing to.  By using this value you can avoid using react `ref`.

```js
handleChange(e) {
  const newValue = e.target.value;
  ...
}
```